src/Microsoft.NetCore.Analyzers/Core/Performance/DoNotUseCountWhenAnyCanBeUsed.cs

```diff
@@ -101,6 +101,13 @@ public sealed class DoNotUseCountWhenAnyCanBeUsedAnalyzer : DiagnosticAnalyzer
     {
         internal const string AsyncRuleId = "CA1828";
         internal const string SyncRuleId = "CA1827";
+        internal const string OperationKey = "Operation";
```

Comment:
Nit: Consider using nameof like `internal const string OperationKey = nameof(OperationKey);` as the actual value is not really important.

Reply:
Same for fields below, if you decide to change.